code/game/objects/items/weapons/powerfist.dm

```diff
@@ -14,10 +14,9 @@
 	origin_tech = "combat=5;powerstorage=3;syndicate=3"
 	var/click_delay = 1.5
 	var/fisto_setting = 1
-	var/gasperfist = 3
+	var/gasperfist = 0.5 //original consumption mod was 3, for reference
```

Comment:
```suggestion
	var/gasperfist = 0.5
```
please dont leave comments like this, we have git history for a reason (this var name is also awful, not your fault though)